src/bsoncxx/oid.hpp

```diff
@@ -38,7 +38,10 @@ BSONCXX_INLINE_NAMESPACE_BEGIN
 class BSONCXX_API oid {
    public:
     struct init_tag_t {};
-    static constexpr init_tag_t init_tag{};
+
+    // TODO(MSVC): Ideally this would be constexpr, but VS2015U1 can't
```

Comment:
It makes it easier to track if the underlying bug has been fixed for someone who comes along later and sees the TODO.